lib/ama_layout/version.rb

```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = "1.0.0"
+  VERSION = "1.0.1"
```

Comment:
@rubene I talked to Mike about this on Friday and he said that whoever get's merged first will use this version number and the other PR will have to update.